src/main/kotlin/com/openlattice/mechanic/upgrades/DeleteAndCreateOrgMetaEntitySets.kt

```diff
@@ -97,17 +97,25 @@ class DeleteAndCreateOrgMetaEntitySets(
 
         tables.keys.forEach { id ->
             val key = AclKey(id)
-            var ids = auditRecordEntitySetsManager.getAuditRecordEntitySets(key)
-            var entitySets = entitySetsService.getEntitySetsAsMap(ids)
-            var propertyTypes = entitySetsService.getPropertyTypesOfEntitySets(ids).mapValues { it.value.values }
-            logger.info("adding audit entity sets associated with org external database tables")
-            metadataEntitySetsService.addDatasetsAndColumns(entitySets.values, propertyTypes)
-
-            ids = auditRecordEntitySetsManager.getAuditEdgeEntitySets(key)
-            entitySets = entitySetsService.getEntitySetsAsMap(ids)
-            propertyTypes = entitySetsService.getPropertyTypesOfEntitySets(ids).mapValues { it.value.values }
-            logger.info("adding audit edge entity sets associated with org external database tables")
-            metadataEntitySetsService.addDatasetsAndColumns(entitySets.values, propertyTypes)
+            try {
```

Comment:
bulking calls and avoiding tons of unnecessary heavy hazelcast calls will make this orders of magnitude faster.

```suggestion
         val entitySetIds = tables.keys.flatMap {
            val aclKey = AclKey(it)
            try {
                auditRecordEntitySetsManager.getAuditRecordEntitySets(aclKey) + auditRecordEntitySetsManager.getAuditEdgeEntitySets(aclKey)
            } catch (e: Exception) {
                logger.error("Unable to load audit entity sets for acl key {}", aclKey)
                setOf<UUID>()
            }
        }

        val entitySets = entitySetIds.mapNotNull { toolbox.entitySets[it] }
        val propertyTypes = entitySets.associate {
            it.id to toolbox.entityTypes.getValue(it.entityTypeId).properties.map { ptId ->
                toolbox.propertyTypes.getValue(ptId)
            }
        }

        logger.info("adding audit entity sets associated with org external database tables")
        metadataEntitySetsService.addDatasetsAndColumns(entitySets, propertyTypes)
```

Reply:
I'm not going to be running `DeleteAndCreateOrgMetaEntitySets`. The `try-catch` change was leftover from before and never got committed.

Reply:
ah cool